lib/internal/crypto/x509.js

```diff
@@ -265,6 +267,21 @@ class X509Certificate {
     return value;
   }
 
+  get extensions() {
+    let extensions = this[kInternalState].get('extensions');
+
+    if (extensions === undefined) {
+      extensions = this.getExtensions();
```

Comment:
Why don't we directly call the underlying implementation rather than exposing an additional function called "getExtensions"?